rpc/core/pipe.go

```diff
@@ -137,7 +135,11 @@ func SetEventBus(b *types.EventBus) {
 }
 
 // SetConfig sets an RPCConfig.
+// If necessary it adjusts (increases) the WriteTimeout for writing to the http
+// connection to ensure rpcserver.WriteTimeout() > c.TimeoutBroadcastTxCommit.
+// For details see: https://github.com/tendermint/tendermint/issues/3435
 func SetConfig(c cfg.RPCConfig) {
+	rpcserver.AdjustWriteTimeout(c)
```

Comment:
move to node.go

Reply:
See https://github.com/tendermint/tendermint/pull/3443#discussion_r266789988

If instead you mean we should move the call of `rpcserver.AdjustWriteTimeout` out of here and into the node package / node.go, we might miss other places which set the config for the rpc package (including the `TimeoutBroadcastTxCommit` which this PR is all about).

Reply:
I understand, but I think it's fine to expect the config to be correct until TimeoutBroadcastTxCommit is further resolved